.github/workflows/deploy-cloudrun-credentials-it.yml

```diff
@@ -1,6 +1,9 @@
 name: deploy-cloudrun credentials Integration
 
-on: [push]
+on:
+  push:
+    branches-ignore: 
+    - 'example-*'
```

Comment:
Do we need to ignore these branches?

Reply:
I think it's nice that if someone is trying to run the example workflows it doesn't kick off this test workflow.